### PR TITLE
Vue 1074 setup and fire the loans shown event on every loan query execution

### PR DESCRIPTION
**Pull request by dyersituations**
https://kiva.atlassian.net/browse/VUE-1074
https://kiva.atlassian.net/browse/VUE-1075
https://kiva.atlassian.net/browse/VUE-1076
https://kiva.atlassian.net/browse/VUE-1077
https://kiva.atlassian.net/browse/VUE-1078
https://kiva.atlassian.net/browse/VUE-1079

- Added loan results/hits analytics
- Added gender filter analytics
- Added sort by analytics
- Added location filter analytics
- Added sector filter analytics
- Added theme filter analytics
- `KvCheckboxList` now emits more change event data to allow analytics to be more detailed

### CODE REVIEW

File: src/components/Lend/LoanSearch/LoanSearchInterface.vue [hunk 1]
```diff
@@ -204,10 +206,27 @@ export default {
...
+
+				// Add analytics event for loans query result
+				this.trackLoans();
 			}
 		});
 	},
 	methods: {
+		trackLoans() {
+			// Check for matching hits, for example when sorting a single page of results
+			const hits = this.loans.map(l => l.id).sort().join();
```
Comment (mcstover) [inline src/components/Lend/LoanSearch/LoanSearchInterface.vue:218]: I think we need to preserve the delivered order of the loan ids. Does this sort alter that?

Comment (dyersituations) [reply src/components/Lend/LoanSearch/LoanSearchInterface.vue:218]: The order delivered to analytics?

Comment (mcstover) [reply src/components/Lend/LoanSearch/LoanSearchInterface.vue:218]: Correct. It should represent the same order in which the loans are shown on the page

Comment (dyersituations) [reply src/components/Lend/LoanSearch/LoanSearchInterface.vue:218]: Do we want to send a new event if the order is different by IDs the same? Previously we checked the IDs to ensure there weren't duplicate lists, regardless of order.

Comment (dyersituations) [reply src/components/Lend/LoanSearch/LoanSearchInterface.vue]: Got it, thanks. I made the false assumption that we wanted to check result duplicates in general. Updated!

**Review by mcstover — APPROVED** [review]

**Review by mcstover — APPROVED** [review]

**Pull request merged**